pyccel/epyccel.py

```diff
@@ -18,10 +18,10 @@
 #==============================================================================
 random_selector = random.SystemRandom()
 
-def random_string( n ):
+def random_string( n, prefix = '' ):
     # we remove uppercase letters because of f2py
     chars    = string.ascii_lowercase + string.digits
-    return ''.join( random_selector.choice( chars ) for _ in range(n) )
+    return prefix + ''.join( random_selector.choice( chars ) for _ in range(n) )
```

Comment:
Was it really necessary to change this function? I see no advantage when you use it.

Reply:
No sorry, I don't remember if this was just to have everything on one line or if I just wanted to be able to determine the whole module name by fiddling with this function while I was debugging